src/modules/Workspaces/WorkspacesEditor/ViewModels/MainViewModel.cs

```diff
@@ -601,5 +603,37 @@ private void SendDeleteTelemetryEvent()
             telemetryEvent.Successful = true;
             PowerToysTelemetry.Log.WriteEvent(telemetryEvent);
         }
+
+        internal static bool IsPwaApp(Models.Application application)
+        {
+            if (application.IsEdge())
+            {
+                return true;
+            }
+            else if (application.IsChrome())
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        internal static string GetPwaIconFilename(Models.Application application)
+        {
+            if (application.IsEdge())
+            {
+                return PwaHelper.GetEdgeAppIconFile(application.PwaAppId);
+            }
+            else if (application.IsChrome())
+            {
+                return PwaHelper.GetChromeAppIconFile(application.PwaAppId);
+            }
+            else
+            {
+                return string.Empty;
+            }
+        }
```

Comment:
It makes more sense to move these functions to the `Application` class because they operate on the internal state of the Application object and are only used within the context of that class. This aligns with the principle of encapsulation and makes the code more cohesive. Furthermore, as I mentioned earlier, it’s a good idea to have a shared `Application` class in the common library, since both `Application` in the Editor and `AppLaunching` in LauncherUI share common functionality. You can implement the specific differences in derived classes, ensuring both reusability and maintainability.